jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1CommandManager.java

```diff
@@ -92,6 +95,14 @@ public void depositToDDS(String asdBytes) throws Asn1CommandManagerException {
 
       HashMap<String, String> responseList = new HashMap<>();
       for (RSU curRsu : request.getRsus()) {
+         
+         if (curRsu.getRsuUsername() == null || curRsu.getRsuUsername().isEmpty()) {
+            curRsu.setRsuUsername(odeProperties.getRsuUsername());
+         }
+         
+         if (curRsu.getRsuPassword() == null || curRsu.getRsuPassword().isEmpty()) {
```

Comment:
SHould this be `curRsu.getRsuPassword() != null && !curRsu.getRsuPassword().isEmpty()`